migrate/scripts/moveExtraStuff.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use CommunityVoices\Model\Mapper;
+use CommunityVoices\Model\Entity;
+
+$quoteMapper = new Mapper\Quote($dbHandler);
+$quoteEntity = new Entity\Quote;
+$allExtraStuffQuery = "SELECT * FROM `community-voices_quotes` WHERE extra_stuff is not null";
+$addEndUseColumnQuery = "ALTER TABLE `community-voices_slides`
+                        ADD COLUMN end_use varchar(255) DEFAULT NULL";
+
+$statement = $dbHandler->prepare($addEndUseColumnQuery);
+$statement->execute(); */
```

Comment:
Does the extra `*/` cause a syntax error?

Reply:
> We have a very quick timeline toward this needing to go live I guess, so I tried to be very lenient with review. I'd say glance at these and let me know if there are any you can do. If not, we can go ahead and merge. It sounds like it is desired that I don't test this (which is part of my lower-quality comments, as I did not run each part of the code to see what it does), and the testing will be done in Cleveland production. Sketchy, but that seems to be the plan.

@Sammidysam I totally agree that is sketchy. I think this calls for a full test, which I expect to pass. Even if you could give 30 minutes for testing that would be very helpful.

Reply:
I will do so.  Could you respond to the original thread starter here though?

Reply:
Fixed